Framework/ServiceClasses/PrinterServiceProvider/Handlers/PrintFormHandler.cs

```diff
@@ -953,15 +953,19 @@ private PrintFormResult CheckFieldValid(FormField field, Form form)
                                                $"COLOR is not valid.",
                                                PrintFormCompletion.PayloadData.ErrorCodeEnum.FieldError);
             }
-            
+
```

Comment:
Thanks - we agree changes. we will merge changes for the printer before next workshop